omim2obo/main.py

```diff
@@ -198,36 +239,49 @@ def omim2obo(use_cache: bool = False):
         #  should pass all symbols/abbrevs from each of preferred, alt, included each time it is called. If no symbols
         #  for given term, should pass empty list. See: https://github.com/monarch-initiative/omim/issues/129
-        abbrev: Union[str, None] = None if not pref_symbols else pref_symbols[0]
+        pref_abbrev: Union[str, None] = None if not pref_symbols else pref_symbols[0]
 
         # Add synonyms
-        graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(pref_title, abbrev))))
-        for alt_label in other_labels:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(alt_label, abbrev))))
-        for abbreviation in pref_symbols:
-            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(abbreviation)))
-            # Reify on abbreviations. See: https://github.com/monarch-initiative/omim/issues/2
-            axiom = BNode()
-            graph.add((axiom, RDF.type, OWL.Axiom))
-            graph.add((axiom, OWL.annotatedSource, omim_uri))
-            graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
-            graph.add((axiom, OWL.annotatedTarget, Literal(abbreviation)))
-            graph.add((axiom, OBOINOWL.hasSynonymType, MONDONS.abbreviation))
-
-        # Add 'included' entry properties
-        included_detected_comment = "This term has one or more labels that end with ', INCLUDED'."
-        if label_endswith_included_alt or label_endswith_included_inc:
-            graph.add((omim_uri, RDFS['comment'], Literal(included_detected_comment)))
-        for included_label in cleaned_inc_labels:
-            graph.add((omim_uri, URIRef(INCLUDED_URI), Literal(label_cleaner.clean(included_label, abbrev))))
+        # - exact titles
+        graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(pref_title, pref_abbrev))))
+        for title in alt_titles:
+            graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(title, pref_abbrev))))
+        # - related titles
+        for title in former_alt_titles:
+            graph.add((omim_uri, oboInOwl.hasRelatedSynonym, Literal(label_cleaner.clean(title, pref_abbrev))))
+        # - exact abbreviations
+        # todo #1: Consider included_symbols for 'exact' list https://github.com/monarch-initiative/omim/issues/140
+        for abbrevs in [pref_symbols, alt_symbols]:
+            for abbreviation in abbrevs:
+                add_triple_and_optional_annotations(graph, omim_uri, oboInOwl.hasExactSynonym, Literal(abbreviation),
+                    [(OBOINOWL.hasSynonymType, MONDONS.abbreviation)])
+        # - related, deprecated 'former' titles
+        for title in former_alt_titles:
+            clean_title = Literal(label_cleaner.clean(title, pref_abbrev))
+            add_triple_and_optional_annotations(graph, omim_uri, oboInOwl.hasRelatedSynonym, clean_title,
+                [(OWL.deprecated, Literal(True))])
```

Comment:
I do not think that the intent was to add owl:deprecated as an annotation on the synonym annotation. I have a question into Sabrina to confirm this

Reply:
Hmm, OK. Right now I can't think of anything else I would deprecate. I wouldn't want to deprecate the whole term, since non-deprecated MIMs can have both deprecated and non-deprecated synonyms.

Reply:
I don't think there is a need to add `owl:deprecated` to anything here

Reply:
Hmm... I'm basing this change off of:
- #138 

In that issue it links to the discussion [with Nicole](https://github.com/monarch-initiative/omim/issues/120#issuecomment-2341484213) where she talks about this being their standard practice.

Example:
> `synonym: "juvenile carcinoma (formerly)" RELATED DEPRECATED [GARD:0009408]`

This change makes it so that this process is automated for them.

Reply:
Ok, let's confirm with @sabrinatoro which of the suggestions to follow since the comments in that thread from Sabrina and Nicole are not in agreement.